sway/commands.c

```diff
@@ -35,6 +35,8 @@
 #include "input.h"
 #include "border.h"
 
+#define ARRAY_SIZE(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
Do you have any other suggestions or do you want me expand the ARRAY_SIZE macro everywhere? If you don't mind what's wrong with having the size constants?